mysql-test/service-test.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
```

Comment:
I like this test a lot! There are various things that we could change, but this sort of script would help make buildbot upgrade tests much more modular. @vladbogo, regardless if this particular file gets into the server, the work here can help when porting upgrade tests from the old buildbot.